tests/acceptance/task-list-test.js

```diff
@@ -46,6 +46,7 @@ test('member can assign/reassign/unassign tasks to user', function(assert) {
   });
 
   andThen(() => {
+    assert.ok(find('.task-list').css('height') > '500px', 'the task-list has height of 100%');
```

Comment:
@JoshSmith not sure I am a huge fan of this...Any other thoughts on testing the height if you still want it tested?